README.md

```diff
@@ -136,9 +131,9 @@ app.on('start', () => {
 -------------------------------------------------------
 <a name="use"></a>
 
-### app.use(func, [opts], [cb])
+### app.use(func, [opts])
```

Comment:
Why did you removed `[cb]`?

Reply:
because it's very bad to use it anyway, but we still support it for backward compat.